src/Core/src/Handlers/RefreshView/RefreshViewHandler.Android.cs

```diff
@@ -66,5 +66,26 @@ public static void MapContent(IRefreshViewHandler handler, IRefreshView refreshV
 
 		public static void MapRefreshColor(IRefreshViewHandler handler, IRefreshView refreshView)
 			=> UpdateRefreshColor(handler);
+
+		public static void MapIsRefreshEnabled(IRefreshViewHandler handler, IRefreshView refreshView)
+			=> UpdateIsRefreshEnabled(handler);
+
+		public static void MapIsEnabled(IRefreshViewHandler handler, IRefreshView refreshView)
+			=> UpdateIsEnabled(handler);
+
+		static void UpdateIsRefreshEnabled(IRefreshViewHandler handler)
+		{
+			handler.PlatformView.Enabled = handler.VirtualView.IsRefreshEnabled;
```

Comment:
I think we may need to add a property to MauiSwipeRefreshLayout.RefreshEnabled, and then have MauiSwipeRefreshLayout.CanChildScrollUp() return true if it is NOT enabled.

The current code here will disable the entire control still as it uses the base Enabled property.